tests/foreman/api/test_contentmanagement.py

```diff
@@ -56,22 +55,22 @@
 from robottelo.helpers import md5_by_url
 from robottelo.host_info import get_repo_files
 from robottelo.host_info import get_repomd_revision
+from robottelo.hosts import Satellite
 from robottelo.utils.issue_handlers import is_open
-from robottelo.vm import VirtualMachine
-from robottelo.vm_capsule import CapsuleVirtualMachine
 
 
-@pytest.fixture(scope='class')
-def capsule_vm():
+@pytest.fixture(scope='function')
+def capsule_vm(capsule_latest):
```

Comment:
This would be great to have under the main broker fixtures!

Reply:
got it. Will make a PR for this.